docs-client/src/containers/App/index.tsx

```diff
@@ -338,13 +346,15 @@ class App extends React.PureComponent<Props, State> {
 
   public componentWillMount() {
     this.initSpecification();
+    this.initVersion();
   }
 
   public render() {
     const { classes } = this.props;
     const { specification } = this.state;
+    const { versions } = this.state;
 
-    if (!specification) {
+    if (!specification || !versions) {
```

Comment:
I guess it would be better to render specifications even if there is issue on versions.  
What do you think about that? 😀

Reply:
I agree. I will change condition :D
Thank you for your advice.